processor/configuration/src/main/resources/schemas/actions/slack_sink_0.1.json

```diff
@@ -20,7 +20,9 @@
           "title": "Webhook URL",
           "description": "The webhook URL used by the Slack channel to handle incoming messages.",
           "type": "string",
-          "format": "password"
+          "format": "password",
+          "$comment": "https://stackoverflow.com/a/6041965/9360757",
+          "pattern": "(http|https):\\\/\\\/([\\w_-]+(?:(?:\\.[\\w_-]+)+))([\\w.,@?^=%&:\\\/~+#-]*[\\w@?^=%&\\\/~+#-])"
```

Comment:
Thinking whether the regex should be tuned up to simplify.
i.e. `(?:` represents a non capturing group which is IMHO not needed in our case.

Reply:
This regex comes from https://stackoverflow.com/a/6041965/9360757 . Do you have a better one that we could use? Btw json schema has a smaller syntax for regex https://json-schema.org/understanding-json-schema/reference/regular_expressions.html